lora-phy/src/sx126x/mod.rs

```diff
@@ -3,13 +3,21 @@ mod radio_kind_params;
 use defmt::debug;
 use embedded_hal_async::delay::DelayNs;
 use embedded_hal_async::spi::*;
+pub use radio_kind_params::Sx126xVariant;
```

Comment:
Put these two on single line?

Reply:
I moved all the variant thing to a separate module. It didn't really belong in radio_kind_params anyway